src/es-module-shims.js

```diff
@@ -52,17 +52,20 @@ async function importMetaResolve (id, parentUrl = this.url) {
   return resolve(id, `${parentUrl}`);
 }
 
-Object.defineProperties(importShim, {
+let esmsState = {}
+self.esmsState = esmsState
+
+Object.defineProperties(esmsState, {
   m: { value: meta },
   l: { value: undefined, writable: true },
   e: { value: undefined, writable: true }
 });
-importShim.fetch = url => fetch(url);
-importShim.skip = /^https?:\/\/(cdn\.pika\.dev|dev\.jspm\.io|jspm\.dev)\//;
-importShim.load = processScripts;
-importShim.onerror = (e) => {
-  throw e;
-};
+
+const esmsInitOptions = self.esmsInitOptions || {};
```

Comment:
Should we delete this global options object after using it?

Reply:
Addressed here: https://github.com/guybedford/es-module-shims/pull/109/commits/b2e8e0b2422b3c77cee8cccadc6e1994d936165d